bootstrapform/templatetags/bootstrap.py

```diff
@@ -77,6 +77,9 @@ def render(element, markup_classes):
             template = get_template("bootstrapform/form.html")
             context = Context({'form': element, 'classes': markup_classes})
 
+        if django_version >= (1, 8):
```

Comment:
Agreed. If the element_type is "boundfield", then the error in the issue description occurs because this fix does not get applied. Would be nice if we could get a new release with this fix.